templates/connect.php

```diff
@@ -422,17 +434,15 @@ class="button button-secondary" tabindex="2"><?php fs_esc_html_echo_x_inline( 'S
 
 			if ( ! empty( $permissions ) ) : ?>
 				<div class="fs-permissions">
-					<?php if ( $require_license_key ) : ?>
-						<p class="fs-license-sync-disclaimer"><?php
-                                echo sprintf(
-									fs_esc_html_inline( 'The %1$s will periodically send %2$s to %3$s for security & feature updates delivery, and license management.', 'license-sync-disclaimer', $slug ),
-									$fs->get_module_label( true ),
-									sprintf('<a class="fs-trigger" href="#" tabindex="1">%s</a>', fs_esc_html_inline('diagnostic data', 'send-data')),
-									'<a class="fs-tooltip-trigger' . (is_rtl() ? ' rtl' : '') . '" href="' . $freemius_site_url . '" target="_blank" rel="noopener" tabindex="1">freemius.com <i class="dashicons dashicons-editor-help" style="text-decoration: none;"><span class="fs-tooltip" style="width: 170px">' . $fs->get_text_inline( 'Freemius is our licensing and software updates engine', 'permissions-extensions_desc' ) . '</span></i></a>'
-								) ?></p>
-					<?php else : ?>
+                    <p class="fs-license-sync-disclaimer"><?php

```

Comment:
@msladu why did you remove the `if`?

Reply:
@malayladu when activating a license key the permissions are triggered from a different link:
![image](https://user-images.githubusercontent.com/6830121/160407006-2074b163-d314-4931-b563-5a47f95cb8f1.png)

Please revert this change.